include/tests_authentication

```diff
@@ -1479,6 +1479,31 @@
     fi
 #
 #################################################################################
+#
+    # Test        : AUTH-9411
+    # Description : Check if /etc/securetty is empty
+    Register --test-no AUTH-9411 --weight L --network NO --category security --description "Checking /etc/securetty file"
+    if [ ${SKIPTEST} -eq 0 ]; then
+        LogText "Test: checking /etc/securetty"
+        if [ -f /etc/securetty ]; then
+            LogText "Result: file /etc/securetty found"
+            FIND=$(${GREPBINARY} -v "^#" '/etc/securetty' | wc -l)
```

Comment:
This count empty lines as well and with only one empty line, this test will always complain. 
It's probably better to use:

```
# use of "^ *" instead of "^\s*", because HPUX und Solaris can't handle "\s". Using " *" seems work on all platforms though
FIND=$(${GREPBINARY} -v "^ *#" '/etc/securetty' | grep -v "^ *$ | wc -l)
```

Reply:
Thank you for review. I'll fix it.